File: Java/Shirt.java
public class Shirt extends Clothing {
    private String color;
    private String sleeveType;

    public Shirt(String color, String sleeveType, char size, String material, char gender, int id, String name, String brand, String price) {
        super(size, material, gender, id, name, brand, price);
        this.color = color;
        this.sleeveType = sleeveType;
    }

    // Other getters and setters
    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getSleeveType() {
        return sleeveType;
    }

    public void setSleeveType(String sleeveType) {
        this.sleeveType = sleeveType;
    }
}

File: Py/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):
    def __init__(self, color="", sleevetype="", size="", materials="", gender=""):
        super().__init__(size, materials, gender)
        self.color = color
        self.sleevetype = sleevetype

    def get_color(self):
        return self.color

    def set_color(self, color):
        self.color = color

    def get_sleevetype(self):
        return self.sleevetype

    def set_sleevetype(self, sleevetype):
        self.sleevetype = sleevetype